modules/administration-guide/pages/repairing-the-devworkspace-operator.adoc

```diff
@@ -0,0 +1,88 @@
+:_content-type: PROCEDURE
+:navtitle: Repairing the {devworkspace} Operator
+:description: Repairing the {devworkspace} Operator
+:keywords: devworkspace, operator, repair
+:page-aliases:
+
+[id="repairing-the-devworkspace-operator_{context}"]
+= Repairing the {devworkspace} Operator
+
+.Prerequisites
+
+* An active `{orch-cli}` session as a cluster administrator to the destination {orch-name} cluster. See {orch-cli-link}.
+
+.Procedure
+
+. Delete the `devworkspace-controller` namespace that contains the failing pod.
+
+. Update `DevWorkspace` and `DevWorkspaceTemplate` CRDs by setting the conversion strategy to `None`:
++
+image::repairing-dwo/looking-up-custom-resource-definitions.png[]
++
+[source,yaml]
+----
+spec:
+  conversion:
+    strategy: None # note .spec.conversion.webhook is removed here
+----
++
+image::repairing-dwo/setting-the-conversion-strategy-to-none.png[]
++
+NOTE: The `DevWorkspaceOperatorConfig` and `DevWorkspaceRouting` CRDs  have the conversion strategy set to `None` by default.
+
+. Remove the {devworkspace} Operator subscription:
++
+[source,terminal]
+----
+$ oc delete sub devworkspace-operator
+----
+
+. Get the {devworkspace} Operator CSVs:
++
+[source,terminal]
+----
+$ oc get csv | grep devworkspace
+----
++
+pass:[<!-- vale CheDocs.Attributes = NO -->]
+.Output displayed in the __<devworkspace-operator.vX.Y.Z>__ format
+====
```

Comment:
Reduce the amount of white space to minimize the probability of the lines wrapping.

Reply:
Done.